pkg/clienttransport.go

```diff
@@ -47,7 +48,20 @@ type ClientTransport struct {
 func (r *ClientTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 	route, pathParams, err := r.Router.FindRoute(req.Method, req.URL)
 	if err != nil {
-		return newError(http.StatusNotFound, err.Error()), nil
```

Comment:
Suggestion: Do you mind documenting the default path logic in the description?

Reply:
Interesting enough, I _very recently_ discussed the need for a "default" backend.  Your timing is coincidentally amazing.

Question though; there _is_ already the ability to add a wildcard path, like so:

```
  /api-docs/{*}:
    get:
      responses:
        '200':
          description: Swagger UI
      x-transportd:
        backend: swaggerui
      tags:
        - Documentation
```

Although the Swagger Editor complains about the `{*}` syntax, it already is processed by transportd correctly.  Could a path like `/{*}` achieve the same goal as introducing the concept of "default backend"?

Reply:
question: Which description would you want this added to? The `ClientTransport`?

Reply:
I believe @iamnotbrandonchan is referring to the Godoc comment for `ClientTransport.RoundTrip`, which currently makes no mention of the new behavior.